packages/climate-ref-pmp/src/climate_ref_pmp/diagnostics/annual_cycle.py

```diff
@@ -32,49 +70,20 @@ class AnnualCycle(CommandLineDiagnostic):
         "statistic",
         "season",
     )
+
     data_requirements = (
-        # Surface temperature
-        (
-            DataRequirement(
-                source_type=SourceDatasetType.PMPClimatology,
-                filters=(FacetFilter(facets={"source_id": ("ERA-5",), "variable_id": ("ts",)}),),
-                group_by=("variable_id", "source_id"),
-            ),
-            DataRequirement(
-                source_type=SourceDatasetType.CMIP6,
-                filters=(
-                    FacetFilter(
-                        facets={
-                            "frequency": "mon",
-                            "experiment_id": ("amip", "historical", "hist-GHG", "piControl"),
-                            "variable_id": ("ts",),
-                        }
-                    ),
-                ),
-                group_by=("variable_id", "source_id", "experiment_id", "member_id", "grid_label"),
-            ),
-        ),
-        # Precipitation
-        (
-            DataRequirement(
-                source_type=SourceDatasetType.PMPClimatology,
-                filters=(FacetFilter(facets={"source_id": ("GPCP-Monthly-3-2",), "variable_id": ("pr",)}),),
-                group_by=("variable_id", "source_id"),
-            ),
-            DataRequirement(
-                source_type=SourceDatasetType.CMIP6,
-                filters=(
-                    FacetFilter(
-                        facets={
-                            "frequency": "mon",
-                            "experiment_id": ("amip", "historical", "hist-GHG", "piControl"),
-                            "variable_id": ("pr",),
-                        }
-                    ),
-                ),
-                group_by=("variable_id", "source_id", "experiment_id", "member_id", "grid_label"),
-            ),
-        ),
+        make_data_requirement("pr", "GPCP-Monthly-3-2"),
+        make_data_requirement("psl", "ERA-5"),
+        make_data_requirement("rlds", "CERES-EBAF-4-2"),
+        make_data_requirement("rlus", "CERES-EBAF-4-2"),
+        make_data_requirement("rlut", "CERES-EBAF-4-2"),
+        make_data_requirement("rsds", "CERES-EBAF-4-2"),
+        make_data_requirement("rsdt", "CERES-EBAF-4-2"),
+        make_data_requirement("rsus", "CERES-EBAF-4-2"),
+        make_data_requirement("rsut", "CERES-EBAF-4-2"),
+        make_data_requirement("ts", "ERA-5"),
+        make_data_requirement("uas", "ERA-5"),
+        make_data_requirement("vas", "ERA-5"),
```

Comment:
Can we do it this way even though it is out of alphabetical order. This will mean we don't need to change any tests as it expects ts/ERA-5 to be the first requirement

```suggestion
        make_data_requirement("ts", "ERA-5"),
        make_data_requirement("uas", "ERA-5"),
        make_data_requirement("vas", "ERA-5"),
        make_data_requirement("psl", "ERA-5"),        
        make_data_requirement("pr", "GPCP-Monthly-3-2"),
        make_data_requirement("rlds", "CERES-EBAF-4-2"),
        make_data_requirement("rlus", "CERES-EBAF-4-2"),
        make_data_requirement("rlut", "CERES-EBAF-4-2"),
        make_data_requirement("rsds", "CERES-EBAF-4-2"),
        make_data_requirement("rsdt", "CERES-EBAF-4-2"),
        make_data_requirement("rsus", "CERES-EBAF-4-2"),
        make_data_requirement("rsut", "CERES-EBAF-4-2"),

```